pkg/server/helpers.go

```diff
@@ -234,9 +229,9 @@ func getRepoDigestAndTag(namedRef reference.Named, digest imagedigest.Digest, sc
 	return repoDigest, repoTag
 }
 
-// localResolve resolves image reference locally and returns corresponding image metadata. It returns
-// nil without error if the reference doesn't exist.
-func (c *criService) localResolve(ctx context.Context, refOrID string) (*imagestore.Image, error) {
+// localResolve resolves image reference locally and returns corresponding image metadata. It
+// store.ErrNotExist if the reference doesn't exist.
```

Comment:
/s/./or docker/distribution/digestset.ErrDigestInvalidFormat, or ErrDigestAmbiguous./

Reply:
Add a todo, we should not be eating the err returned from util.NormalizeImageRef. We are currently assuming it must be a digest if it's not a valid image ref.. but it could just be a poorly formed image ref and not a digest. But right now if it's a poorly formed image ref we return the error from digest lookup.  Let's keep "both" errors and combine them here, so the user can see it's not a valid digest or image reference.

Reply:
Done